Components/cards/cards.css

```diff
@@ -80,4 +84,11 @@ header {
 
 .card-with-shadow {
     box-shadow: 1rem 1rem var(--secondary);
+}
+
+.badge-text{
+    position: absolute;
+    background-color: var(--primary);
+    padding: 2px;
+    color: var(--white);
```

Comment:
avoid defining variable name same as color name
white:white?? why

Reply:
Yeah, just used the variables from the initial setup